### ISSUE TITLE
Search Box - UX and Accessibility Improvements

### ISSUE THREAD

**Issue by bnhowell** [mlibrary/fishrappr#131]
## Description: 
As a user I want to be able to search the Michigan Daily Archive from the home page and search results page using keywords(booleans, exclusions and wildcards) with/without a specific date (yyyy, yyyy/mm or yyyy/mm/dd) or date range.  

## Acceptance Criteria:
- [x] 1. A responsive search box 
- [x] 2. Assistive technology will describe group as a search box and speak the name of form elements and controls within the box
- [ ] 3. Visual styling of search box to align more with blacklight palette in Home and Search Pages
For home page search box styling see: [](https://invis.io/VS73TWYC2#/171013815_Home_Page_Light_Search_Box)
For search results search box styling see: [](https://invis.io/VS73TWYC2#/171024410_Search_Results_-_New_Search_Bar)
- [x] 4. Use of tabs to toggle between specific date search and date range search
- [x] 5. Default Date Selections are:  Any Month (Dropdown), Any Day (Dropdown) and YYYY (Field Entry)

**Comment by kaylaondracek** [comment]
Closing - will create new ticket to reflect recent changes to search design
